django/chat/views.py

```diff
@@ -642,14 +642,6 @@ def _copy_options(source_options, target_options):
         chat_options = chat.options
         post_data = request.POST.copy()
 
-        # In case of duplicate values, remove them by taking the first value from each list
```

Comment:
Why remove this, out of curiosity?

Reply:
I honestly don't know why this was in here to begin with.
It breaks the saving of anything with multiple values! (specifically, the "data sources", "documents" multiselects).